src/main/java/com/stripe/model/ExpandableFieldDeserializer.java

```diff
@@ -0,0 +1,45 @@
+package com.stripe.model;
+
+import com.google.gson.*;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+
+public class ExpandableFieldDeserializer implements JsonDeserializer<ExpandableField> {
+    public ExpandableField deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException {
+        Gson gson = new GsonBuilder()
+                .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+                .create();
+
+        if (json.isJsonNull()) {
+            return null;
+        }
+
+        ExpandableField expandableField;
+
+        // Check if json is a String ID:
```

Comment:
Could you make this comment a little more elaborate? e.g. "Check if the JSON data is a string ID. If it is, then that's an indication that this field has not been expanded."